examples/pinterest.amp.html

```diff
@@ -0,0 +1,480 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Pinterest examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script custom-element="amp-pinterest" src="../dist/v0/amp-pinterest-0.1.max.js" async></script>
+  <script src="../dist/amp.js" async ></script>
```

Comment:
haha no need i think. [`async` support](http://caniuse.com/#feat=script-async) is pretty good at this point